### PR TITLE
Add --theme argument and update readme accordingly

### PR DESCRIPTION
**Pull request by shelt**
This commit adds a --theme argument as well as a variable page_path which concatenates the argument into a path. Nonexistent paths are also checked for.

### CODE REVIEW

File: wifiphisher.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
 PORT = 8080
 SSL_PORT = 443
 PEM = 'cert/server.pem'
-PHISING_PAGE = "access-point-pages/minimal"
```
Comment (sophron) [inline wifiphisher.py]: There's still a hard-coded value and that's "access-point-pages/". You just need to rename the variable to "PHISHING_PAGES_DIR".

File: wifiphisher.py [hunk 2]
```diff
@@ -681,6 +680,11 @@ def get_hostapd():
     get_hostapd()
     # Parse args
     args = parse_args()
+    
+    # Verify the specified theme exists
+    page_path = 'access-point-pages/' + str(args.theme) + '/index.html'
```
Comment (sophron) [inline wifiphisher.py]: And this should now be: PHISHING_PAGES_DIR + str(args.theme) + '/index.html'

**Comment by sophron** [comment]
Thanks @shelt for all the work you are doing.

**Comment by shelt** [comment]
@sophron Done.

**Comment by sophron** [comment]
@shelt Some conflicts have occurred with this PR. Can you resolve them?

**Comment by shelt** [comment]
@sophron Done.

**Comment by shelt** [comment]
@sophron Done.

**Comment by sophron** [comment]
Hi @shelt,

Have you tested this PR? I just tried it on my machine and when i hit http://127.0.0.1 it wouldn't load all the necessary files.

**Comment by sophron** [comment]
Done in #208.

**Pull request closed without merging**